packages/fela/src/__tests__/combineRules-test.js

```diff
@@ -70,4 +70,31 @@ describe('Combining rules', () => {
       padding: 20,
     })
   })
+
+  it.only('should merge _className as a special key', () => {
```

Comment:
```suggestion
  it('should merge _className as a special key', () => {
```

oops

Reply:
Ops indeed always doing that same mistake again and again